app/src/main/res/layout/no_collections_message.xml

```diff
@@ -21,7 +21,6 @@
         android:text="@string/collections_header"
         android:textAppearance="@style/HeaderTextStyle"
         android:textSize="16sp"
-        app:fontFamily="@font/metropolis_semibold"
```

Comment:
Same question as above

Reply:
Applied `HeaderTextStyle` contains the font weight.